src/test/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImplTest.java

```diff
@@ -63,6 +101,43 @@ public void tearDown() throws Exception {
         cuExtractPaymentServiceImpl = null;
         FileUtils.forceDelete(new File(ACH_BATCH_TEST_FILE_PATH).getAbsoluteFile());
     }
+    
+    @Test
+    public void testDvCheckStubCreation() {
+        int line1ArrayPosition = 0;
```

Comment:
Adjusted unit test to more closely reflect the code being actually run by the batch job to ensure logic was being adequately unit tested.